addons/block_code/ui/picker/picker.gd

```diff
@@ -25,7 +25,7 @@ func bsd_selected(bsd: BlockScriptData):
 
 	var parent_class: String
 	if found_simple_class_script:
-		parent_class = found_simple_class_script.get_base_class()
+		parent_class = str(found_simple_class_script.get_instance_base_type())
```

Comment:
Can you explain this change? What is this doing and why is it needed? Ideally in each commit message.

Reply:
This was explained in #59.